docs/dev/Lua API.rst

```diff
@@ -614,6 +614,10 @@ Miscellaneous
   both from the curry call and the closure call itself. I.e.
   ``curry(func,a,b)(c,d)`` equals ``func(a,b,c,d)``.
 
+* ``dfhack.msize(ptr)``
```

Comment:
if we expose this, I think it should be in the `dfhack.internal` namespace since we do not expect end user scripts to be calling it

Reply:
Will do. I'm going to rework this PR because as-is its insufficient for a lot of things we'd want to do with it, so I'll give you a ping